web/src/test/java/org/cbioportal/genome_nexus/web/validation/GenomicLocationValidatorTest.java

```diff
@@ -0,0 +1,33 @@
+package org.cbioportal.genome_nexus.web.validation;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+public class GenomicLocationValidatorTest
+{
+    @Test
+    public void testMyVariantInfoAnnotation()
+    {
+        String[] genomicLocation = {
+            "7,140453136,140453136,A,ATGC-T", // true
+            "A,37880220,37880220,T,C", // false
+            "7,-1,37879794,G,C", // false
+            "12,25398284,-2,CC,GA", // false
+            "12,25380271,25380271,12,T", // false
+            "10,89692940,89692940,C,Q", // false
+            "X,89692940,89692940,C," // false
```

Comment:
Let's add one more test for malformed input, something like `X_89692940_89692940_C_T`